tac-operation-lifecycle/tac-operation-lifecycle-server/src/services/operations.rs

```diff
@@ -99,69 +99,75 @@ impl TacService for OperationsService {
     ) -> std::result::Result<tonic::Response<OperationsResponse>, tonic::Status> {
         let inner = request.into_inner();
 
-        match self
-            .db
-            .get_operations(PAGE_SIZE, inner.page_token, OrderDirection::LatestFirst)
-            .await
-        {
-            Ok(operations) => {
-                let last_timestamp = operations
-                    .last()
-                    .map(|op| op.timestamp.and_utc().timestamp() as u64);
-
-                Ok(tonic::Response::new(OperationsResponse {
-                    items: OperationsService::convert_short_db_operation_into_response(operations),
-                    next_page_params: last_timestamp.map(|ts| Pagination {
-                        page_token: ts,
-                        page_items: inner.page_items.unwrap_or(0) as u32 + PAGE_SIZE as u32,
-                    }),
-                }))
+        match inner.q {
+            Some(q) => {
```

Comment:
move this branch to separate function, such that this main function will be like

```
match inner.q {
   Some(query) => get_operations_with_query(),
   None => get_operations(),
}
```

Reply:
`impl TacService for OperationsService` should be as short as possible

Reply:
Done in https://github.com/blockscout/blockscout-rs/pull/1368/commits/cf7ba5bd167060b9509b6d8b616ede1defa5bdd6 by moving multisearch routine into the logic module (`database.rs`)